index.rst

```diff
@@ -147,176 +246,121 @@ The white light source and monochromator sit on a table surrounded by sturdy bla
+         :alt: step_2
+
+#. Remove the linear actuator and place it out of the way
 
-#. Open slits on monochromator
-#. Confirm that remote control is selected on WLS
+      .. figure:: /_static/step_3.png
+         :name: step_3
+         :target: ../_images/step_3.png
+         :alt: step_3
 
-Now you are ready to start turning things on and testing things. 
+#. Remove the final bolt on the shutter and remove the shutter
 
-Operational Setup
------------------
-First, addresses for all of the control components will have to be established. That is being tracked in JIRA ticket `IT-4096 <https://jira.lsstcorp.org/browse/IT-4096>`__. 
+      .. figure:: /_static/step_4.png
+         :name: step_4
+         :target: ../_images/step_4.png
+         :alt: step_4
 
-Make sure that the indexes for each item is correct (for the CSC) and override yaml files are defined and identified.
+#. If you need to open the center door, remove the "closed" limit switch
 
-Then, go to the address for the PDU and confirm that the numbering of the outlets agrees with the table above and what is actually implemented.
+      .. figure:: /_static/step_5.png
+         :name: step_5
+         :target: ../_images/step_5.png
+         :scale: 50 %
 
-You can now power everything up. Initialize the monochromator by logging into the windows machine via remote desktop.
+
+Chiller Maintenance
+-------------------
+
+You may need to refill the water in the chiller occassionally. This should be done with distilled water mixed with 10% of methanol.
+
+.. figure:: /_static/fill_cover.png
+   :name: fill_cover
+   :target: ../_images/fill_cover.png
+   :alt: fill_cover
+   :scale: 50 %
+
+   Chiller cap for refilling water/methanol mixture.
+
+Using Fiber Illuminator 
+-----------------------
+You might want to use the fiber illuminator to calibrate the fiber spectrograph. This spectral calibration lamp will be kept in storage until needed. When you want to use it, follow these steps:
+
+#. Open the right door 
+#. Plug in the fiber illuminator to power (12V). The power cable should be sitting right next to the monochromator
+#. Remove the end of the fiber from its place in the optical path and install it into the fiber illuminator 
+
+   .. note:: You may actually want to install an absorber between the lamp and the fiber spectrograph
+
+#. Turn on the illuminator and take spectra with the fiber spectrograph
+
+.. figure:: /_static/fib_lamp.png
+   :name: fib_lamp
+   :target: ../_images/fib_lamp.png
+   :alt: fib_lamp
+
+   Fiber Illuminator, which will stay in storage until needed.
+
+Monochromator Initialization
+----------------------------
+
+When the system, inlcuding the monochromator windows machine, is power cycled, you need to Initialize the monochromator before using. This can be done using Remote Desktop using the IP address listed above for the monochromator. The username is **MONOCHROMATOR-P\Monochromator**. Ask Patrick Ingraham or Parker Fagrelius for the password. 
```

Comment:
You can just say the creds are in the AuxTel 1password vault.

Reply:
done